posthog/models/filters/mixins/funnel.py

```diff
@@ -1,10 +1,33 @@
 from typing import Optional
 
-from posthog.constants import FUNNEL_STEP, FUNNEL_WINDOW_DAYS
+from posthog.constants import FUNNEL_FROM_STEP, FUNNEL_STEP, FUNNEL_TO_STEP, FUNNEL_WINDOW_DAYS
 from posthog.models.filters.mixins.base import BaseParamMixin
 from posthog.models.filters.mixins.utils import cached_property, include_dict
 
 
+class FunnelFromToStepsMixin(BaseParamMixin):
```

Comment:
Why is this repeated in another file

Reply:
Merge conflict solving gone wrong! Fixed